types/react-places-autocomplete/index.d.ts

```diff
@@ -4,8 +4,9 @@
 //                 Andrew Makarov <https://github.com/r3nya>
 //                 Nokky Goren <https://github.com/ApeNox>
 //                 Aziz Khambati <https://github.com/azizhk>
+//                 Mark Nelissen <https://github.com/marknelissen>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.7
```

Comment:
Hi! Please revert this line. This is not necessary for this PR.

Reply:
Actually, it is. Without the Typescript version update, the CI build fails. It's the error of the second commit. The error is in the googlemaps typings dependency.
[Failed Travis build](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/630561537?utm_source=github_status&utm_medium=notification)

![afbeelding](https://user-images.githubusercontent.com/10599177/71564289-b3534680-2a9e-11ea-9a16-e25e2de4a0f8.png)

Reply:
Oh, I see.

I've checked types for googlemaps -> https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/googlemaps/index.d.ts#L19

It use typescript 3.0 version. Can you bump TS version to 3.0 please? :)

Reply:
Done